### ISSUE TITLE
Witcher 2 crashes  randomly or on saved game delete

### ISSUE THREAD

**Issue by hyperdrive** [virtual-programming/witcher2-linux#157]
Hi,

The game seems to be accumulating corruption as the story progresses. In the beginning I had no issues even without the linux beta patch. A few hours later the game crashes at almost any event.

After reading through the bug reports below I noted that it might be correlated to the number of saved games. As I attempted to delete previous saves I noted that this is a quick way to crash the application. Hence this is a bug report for a complete game crash if you delete enough saved games. It sounds like it should be an easy fix, right? ;-)

cheers,
Kristofer
Ubuntu 14.04 LTS - nvidia 980 TI - 8GB RAM

**Comment by licaon-kter** [comment]
/close as duplicate of https://github.com/virtual-programming/witcher2-linux/issues/108

TL;DR: initial game issue on Windows too, no access to source code means no fix possible. Delete (backup) savegames in your file manager, not in game.
